Разметка внутренней страницы

Ierixorn opened the pull request:
Полная разметка каталога согласно макету

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1898375/tasks/4)

Ierixorn committed:
Размечен каталог

Полная разметка каталога согласно макету

Ierixorn committed:
Исправления ошибок (главная и каталог)

Исправления ошибок на главной и в каталоге с учётом комментариев (Разметка главной страницы).

sealeye commented on site_pages/catalog.html:
```diff
@@ -0,0 +1,323 @@
+        <section class="catalog-filters">
+            <h2 class="">Фильтры каталога</h2>
+            <form class="catalog-filter-form" action="https://echo.htmlacademy.ru/" method="get">
+                <span class="filter-description">Сортировка:</span>
```
Это скорее заголовок. 
Либо legend, если использовать fieldset.

fieldset очень желательно использовать для группировки элементов формы

sealeye reviewed (commented):
В целом, всё ок. Но нужно поработать над формой и использовать legend и fieldset.

Ierixorn committed:
Изменения формы фильтров

Решил использовать вместо span заголовок h3 во всех элементах формы и заключить объединить чекбоксы в fieldset скрыв legend.

sealeye commented on site_pages/catalog.html:
```diff
@@ -0,0 +1,326 @@
+        <section class="catalog-filters">
+            <h2 class="">Фильтры каталога</h2>
+            <form class="catalog-filter-form" action="https://echo.htmlacademy.ru/" method="get">
+                <h3 class="filter-title">Сортировка:</h3>
```
Тут тоже может быть legend и объединить заголовок с селектом

sealeye commented on site_pages/catalog.html:
```diff
@@ -0,0 +1,326 @@
+                    <option value="ascending">По возрастанию цены</option>
+                    <option value="descending">По убыванию цены</option>
+                    </select>
+                <h3 class="filter-title">Цена: 100 ₽ – 500 ₽</h3>
```
Тут тоже лучше использовать fieldset с legend
